src/services/MailService.js

```diff
@@ -54,10 +51,15 @@ export default class MailService {
   async sendConfirmationMail (offer: Offer, token: string): Promise<void> {
     const subject = 'Informationen zu Ihrem Wohnungsangebot'
     const expirationDate = getFormattedDate(offer.expirationDate)
-    const deletionUrl = getDeletionUrl(hostname, token)
-    const extensionUrl = getExtensionUrl(hostname, token)
-
-    const html = renderConfirmationMail({expirationDate, deletionUrl, extensionUrl, portalName})
+    const deletionUrl = getDeletionUrl(cityConfigs[offer.city].hostName, token)
+    const extensionUrl = getExtensionUrl(cityConfigs[offer.city].hostName, token)
+
+    const html = renderConfirmationMail({
```

Comment:
Can we make here sure that javascript is escaped? Or is this the default?

Reply:
Don't know, also not in scope. Maybe create a research task :thinking:

Reply:
But in this case also not necessary, because there's no "user inputted value", right?